### ISSUE TITLE
use $VCF_FILE environment variable as default

### ISSUE THREAD

**Issue by linuxcaffe** [l0b0/vcf#3]
If vcf looked for a $VCF_FILE envar, then the source-file/folder could be omitted.

<pre>
./vcf.sh jane ~/contacts/*.vcf
</pre>

becomes (after putting it in the $PATH)

<pre>
vcf jane
</pre>

Other *.vcf files/folders could still be queried by including the path/to/other.vcf

**Comment by l0b0** [comment]
tl;dr Not gonna happen, I'm afraid.

Since [globbing is a _shell_ expansion](http://mywiki.wooledge.org/glob), that would not work as you expect. For example, if you declared `VCF_FILE='~/contacts/*.vcf'` every *nix tool would look for the _literal single file path_ `~/contacts/*.vcf`, which is a valid path. And there would literally be no way to pass multiple filenames without inventing a custom escaping or quoting scheme, since POSIX doesn't support array variables. Declaring `VCF_FILE=~/contacts/*.vcf` (without the [quotes](http://mywiki.wooledge.org/Quotes)) _would_ expand the tilde and glob, but you can either treat that as one filename (like almost all *nix tools do) or as multiple filenames, at which point you can no longer support filenames containing spaces.

See [Greg's Wiki article on arguments](http://mywiki.wooledge.org/Arguments) for more.

**Comment by linuxcaffe** [comment]
thanks for the explanation and the link!
